src/main/java/by/itechart/api/exception/UserResponseEntityExceptionHandler.java

```diff
@@ -26,6 +26,14 @@ protected ExceptionInfo handleUserNotFoundException(UserNotFoundException ex) {
         return new ExceptionInfo(bodyOfResponse);
     }
 
+    @ResponseBody
+    @ExceptionHandler(UserEmailDuplicationException.class)
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    protected ExceptionInfo handleUserDuplicationException(UserEmailDuplicationException ex) {
+        String bodyOfResponse = ex.getMessage();
+        return new ExceptionInfo(bodyOfResponse);
```

Comment:
Just a nitpicking: this can be a one-liner
```suggestion
        return new ExceptionInfo(ex.getMessage());
```